openmdao/utils/testing_utils.py

```diff
@@ -299,6 +299,86 @@ def force_check_partials(prob, *args, **kwargs):
     return prob.check_partials(*args, **kwargs)
 
 
+def _fix_comp_check_data(data):
+    """
+    Modify the data dict to match the problem format if there is only one fd step size.
+
+    Parameters
+    ----------
+    data : dict
+        Dictionary containing derivative information keyed by subjac.
+    """
+    names = ['J_fd', 'abs error', 'rel error', 'magnitude', 'directional_fd_fwd',
+             'directional_fd_rev']
+
+    for name in names:
+        if name in data:
+            data[name] = data[name][0]
+    if 'steps' in data:
+        del data['steps']
+
+
+def compare_prob_vs_comp_check_partials(probdata, compdata, comp):
```

Comment:
I'm not finding any usages of this function.  Is this only necessary for debugging of this PR?

Reply:
it's used in some tests